python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1549,10 +1613,161 @@ def sqrt(x, out=None, **kwargs):
         square-root of each element in `x`. This is a scalar if `x` is a scalar.
 
     Notes
-    ----
+    -----
     This function only supports input type of float.
     """
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
 
 
+@set_module('mxnet.symbol.numpy')
+def ceil(x, out=None, **kwargs):
+    r"""
+    Return the ceiling of the input, element-wise.
+
+    The ceil of the ndarray `x` is the smallest integer `i`, such that
+    `i >= x`.  It is often denoted as :math:`\lceil x \rceil`.
+
+    Parameters
+    ----------
+    x : _Symbol or scalar
+        Input array.
+    out : _Symbol or None
+        A location into which the result is stored. If provided, it
```

Comment:
Based on templates:
out : _Symbol or None
        Dummy parameter to keep the consistency with the ndarray counterpart.